draft-ietf-oauth-device-flow.xml

```diff
@@ -636,6 +636,14 @@
           from. Thus, signals from the approving user's session and device are
           not relevant to the trustworthiness of the client device.
           </t>
+	  <t>
+	    Note that if an authorization server used with this flow is malicious,
+	    then it could man-in-the middle the backchannel flow to another authorization server.
+	    For this to be possible, the device manufacturer must either be the attacker
```

Comment:
Regarding "the device manufacturer must either be the attacker" are you documenting a possibility where an honest app is running on an evil operating system, which is doing the MITM?